### PR TITLE
Add read file example for try

### LINKED ISSUE #32505: Update `try/catch` docstring to include assigning result to variable
It might be useful to update the `try/catch` docs to include an example of assigning the result of the `try` block to a variable, e.g.:

```julia
lines = try
    open("/danger", "r") do f
        readlines(f)
    end
catch
    @warn "File not found."
end
```

https://github.com/JuliaLang/julia/blob/55e36cc308b66d3472990a06b2797f9f9154ea0a/base/docs/basedocs.jl#L572-L578

### PR DESCRIPTION
**Pull request by Moelf**
fixes #32505

### CODE REVIEW

File: base/docs/basedocs.jl [hunk 1]
```diff
@@ -600,6 +600,16 @@ catch
     @warn "Could not write file."
 end
 ```
+or, when the file cannot be read:
+```julia
+try
```
Comment (harryscholes) [inline base/docs/basedocs.jl]: Assign the `try` block to a variable

Comment (Moelf) [reply base/docs/basedocs.jl]: good point

Comment (harryscholes) [reply base/docs/basedocs.jl]: Cool. Also can you add blank lines before and after the code blocks?

File: base/docs/basedocs.jl [hunk 2]
```diff
@@ -600,6 +600,16 @@ catch
     @warn "Could not write file."
 end
 ```
```
Comment (harryscholes) [inline base/docs/basedocs.jl]: Can you add blank lines before and after the code blocks?

Comment (Moelf) [reply base/docs/basedocs.jl]: yes

**Comment by kshyatt** [comment]
What's the status on this?

File: base/docs/basedocs.jl [hunk 3]
```diff
@@ -601,6 +601,7 @@ catch
...
```
Comment (harryscholes) [inline base/docs/basedocs.jl]: There should be a blank line below this code block as well

**Comment by harryscholes** [comment]
I'm keen to get this merged. Only the whitespace needs to be fixed, then it's good to go

**Comment by Moelf** [comment]
done, thanks for the comment, I sometimes overlook formattings, something I need to work on especially when I'm contributing to docs.

**Pull request merged**